src/arcade_collection/output/parse_growth_file.py

```diff
@@ -0,0 +1,143 @@
+import json
+from os import path
+import tarfile
+from typing import List, Union
+
+import ntpath
+import numpy as np
+import pandas as pd
+from prefect import task
+
+
+GROWTH_COLUMNS = [
+    "TICK",
+    "SEED",
+    "U",
+    "V",
+    "W",
+    "Z",
+    "POSITION",
+    "POPULATION",
+    "STATE",
+    "VOLUME",
+    "CYCLE",
+]
+
+
+@task
+def parse_growth_file(tar: tarfile.TarFile) -> pd.DataFrame:
+    """
+    Parse the tumor growth tar file.
+
+    Parameters
+    ----------
+    tar :
+        Tar file of simulations.
+
+    Returns
+    -------
+    :
+        Data of all timepoints of all simulations in tar file.
+    """
+    all_timepoints = []
+    for member in tar.getmembers():
+        extracted_member = tar.extractfile(member)
+        if extracted_member is not None:
+            base = ntpath.basename(member.name)
+            base_file = path.splitext(base)
+            file_name = base_file[0]
+            extension = base_file[1]
+            if file_name[0] != "." and extension == ".json":
+                extracted_json = json.loads(extracted_member.read().decode("utf-8"))
+                seed = extracted_json["seed"]
+
+                for timepoint in extracted_json["timepoints"]:
```

Comment:
This block can be simplified into a list comprehension (see the `parse_cell_file.py` as an example)